handbook/ce/ce_to_eng_handover.md

```diff
@@ -0,0 +1,35 @@
+# CE-to-Engineering issue handover process
+
+When a prospect or customer files a new support ticket or raises a new issue, CE and Engineering should follow the steps below to ensure the issue is handled successfully.
+
+## Principles
+
+**CE is always the first line of defense.**
+
+**Engineering should only feel a responsibility to get involved if tagged in by CE.**
+
+**However, once someone [takes ownership](support.md#support-owners) of a ticket (whether formally, or informally by taking over the conversation), it is their responsibility to see it through, either by (1) see the issue through to resolution or (2) assign a new owner.**
+
+Exceptions to the principles above:
+
+- This does not apply to our public GitHub issue tracker; instead, it only applies to [official support channels](support.md). Issues filed in GitHub are the responsibility of Engineering and/or Product.
+- Certain customers pay for dedicated support from a member of the Engineering team. Responding to issues filed by these customers is a shared responsibility for the assigned Engineer and CE (whoever sees it first).
+- If an engineer sees a new question or issue come in from a company that they've already been introduced to, or if the question is in their direct area of expertise, they are encouraged to jump in directly.
```

Comment:
An engineer should always check in with a CE before replying if it's not a conversation they're already having (but a new thread or ticket). If it's an existing thread the engineer should be the one to triage and either continue responding or let the CE owner know that despite it being the same ticket or thread, it's a new request, and they'd like to hand it back over to the CE for ownership.